raccoochat/RaccooChatServer.cpp

```diff
@@ -1,27 +1,18 @@
+#include "RaccooChatDB.cpp"
+#include "RaccooChatDB.h"
 #include "RaccooChatServer.h"
 
 #include <algorithm>
+#include <ctime>
 #include <fstream>
-#include <map>
-#include <vector>
 
 #include <glog/logging.h>
 
 namespace raccoochat {
 
-raccoochat::User RaccooChatHandler::createNewUser() {
-  User newUser;
-  newUser.userId = raccooUserId_++;
-  newUser.allMessageIndex = chatMessages_.size() - 1;
-  std::vector<raccoochat::SimpleMessage> tmpMessages;
-  privateMessages_[newUser.userId] = tmpMessages;
-  newUser.privateMessageIndex = -1;
-  return newUser;
-}
-
 std::string RaccooChatHandler::codePassword(const std::string& userPassword) {
   std::string newPassword;
-  const char secret = '!';
+  const char& secret = '!';
```

Comment:
you don't need a reference here.

also if you want to have a constant, you should put it in anon namespace outside of the raccoochat namespace, like this

namespace {
  const char kSecretChar = '!';
}

Note the prefix 'k' for the constants.